doc/maintainers/team_process.md

```diff
@@ -180,6 +180,21 @@ now the most important thing is to direct users to a single destination.*
 
 ## Old issue closing policy
 
-We have yet to settle on a precise guide for closing out old issues, but, for
-now, if you happen across something that could easily be marked as resolved,
-please do so.
+A project like Stack can have a tendency to acquire issues over time, which can
+include:
+
+* Feature requests that won't be included
+* Support requests which have either been answered or stalled
+* Legitimate bugs reports to be fixed and features to be implemented
+
+Unfortunately, over time the signal-to-noise ratio can make it difficult to
+focus on the highest value issues. This makes it difficult for maintainers to
+focus their attention, or new contributors to find a good place to start.
```

Comment:
The separate task list (which may even be based on lists of issues) should help here. I'd argue it's orthogonal to automatically closing issues.